src/components/Header.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable react/no-find-dom-node */
+import React from 'react';
+import { ErrorMessages } from '../types/ErrorMessages';
+import { USER_ID, addTodo } from '../api/todos';
+import { Todo } from '../types/Todo';
+import { useTodos } from '../utils/TodoContext';
+import classNames from 'classnames';
+
+export const Header: React.FC = () => {
+  const {
+    isLoading,
+    setIsLoading,
+    setTodos,
+    tempTodo,
+    setTempTodo,
+    title,
+    setTitle,
+    showError,
+    todos,
+  } = useTodos();
+
+  const userId = USER_ID;
+  const completed = false;
```

Comment:
```suggestion
  const isCompleted = false;
```

let's use `is/has` prefix for boolean variable